### ISSUE TITLE
2 swaps have failed in a row.. Cant proceed

### ISSUE THREAD

**Issue by Cokeman60** [UnstoppableSwap/unstoppableswap-gui#171]
## Expected Behavior

Swap should move to next step, "They lock in their XMR"
## Current Behavior
Sitting at "Locking in your BTC" with over 15 confirmations. Big yellow warning banner. 2nd attempt. 

No clue what's wrong, tried 2 separate exchange providers so can't be that

(was working last week, now can't complete a swap......)
(ps, 50/50 on failing to Redeem xmr, but using "swap.exe" i can retrieve xmr)
## Steps to Reproduce

1. Fill out Monero send

2. Fill out BTC Refund address

3. Send BTC to provided BTC address

4. Initiate swap.... an wait for nothing except refund timer

## Your Environment

- Operating System and version : Windows 11 and v0.5.2

**Comment by binarybaron** [comment]
Would you mind sharing your swap-ids with me? I might be able to correlate them to activity on my swap provider if you swapped with me.

**Comment by Cokeman60** [comment]
Why?

**Comment by Cokeman60** [comment]
"swap_id":"e5f3d9a7-3874-42d1-bfd8-dfb4736944f4"

**Comment by Cokeman60** [comment]
????????

**Comment by binarybaron** [comment]
This is due to the Bitcoin mempool being so filled up. The fees for the BTC redeem transaction which were negotiated at the start of the swap are now too low to even be included in the mempool. I have opened an issue for this here https://github.com/comit-network/xmr-btc-swap/issues/1453


Were you able to refund?

**Comment by binarybaron** [comment]
This might be due to the Bitcoin mempool being so filled up. The fees for the BTC redeem transaction which were negotiated at the start of the swap are now too low to even be included in the mempool. I have opened an issue for this here https://github.com/comit-network/xmr-btc-swap/issues/1453


Were you able to refund?

**Comment by Cokeman60** [comment]
Ah okay so your answer lines up with the solution i had figured out. I wasn't aware there was essentially a minimum transaction amount and having very low fees selected in my wallet, would make my balance say unconfirmed. 

I essentially had to learn about Child Pays For Parent (CPFP) and how to apply it. Fixed old transaction, new transactions also helped move things to confirm.

Yes i still got my funds 0 issues there. Just like ate like $20 out of the $50
In my opinion do not try to swap anything below idk $75 USD lol

**Comment by Cokeman60** [comment]
1st swap today 0 issues

Currently doing a 2nd swap (over $80 USD) it's stuck "locking in BTC" again. Has Over the required 2 confirmations... This time sent funds straight from Coinbase to swap BTC address

**Comment by Cokeman60** [comment]
weirdly both times has cleared the swap dialog box and says 0 providers available... But the second i click History, it immediately pops back up and resumes.... Without me clicking anything

**Comment by Cokeman60** [comment]
"waiting for Alice to lock Monero"
"Connected to Alice at /ip4/45."blah.blah ""
10min ago per the log

**Comment by Cokeman60** [comment]
Confirm or Deny

Buy BTC on coinbase, send To provided Swap BTC address or best practice to just eat the fees and send From coinbase To private wallet/s then From private wallet To provided swap BTC Address???????

**Comment by Cokeman60** [comment]
New day for new swaps... And failed..............

I'm bout to uninstall. You'd throw away a lightbulb that only works 20% of the time...... So why would i keep using this hit or miss tool..... Sick of wasting time.

**Comment by binarybaron** [comment]
These failed swaps were with /ip4/45.138.50.29/tcp/9939/p2p/12D3KooWMgGjeW7ErQxCQzaeHiXxJn42wegCPFepixEXfBJT1PNS? I'll try to get in touch with them to find out what could be the issue here.

**Comment by Cokeman60** [comment]
Only 1 provider now.. Should i be concerned??

**Comment by binarybaron** [comment]
No, nothing to be concerned about.

**Comment by Cokeman60** [comment]
Now if swaps would just work

I love this tool, i want to use it. I hope we can sort it

**Comment by Cokeman60** [comment]
i'm a lil salty.. After 3 failed swaps your tool has eaten $50 in crypto.... Cmon man for failed transactions that's too much......

**Comment by binarybaron** [comment]
I'd advise you to stop initiating new swaps until this issue is fixed.

**Comment by Cokeman60** [comment]
I meant since messaging you, i haven't use your app since lol.
Any word from that malicious provider?

**Comment by binarybaron** [comment]
> weirdly both times has cleared the swap dialog box and says 0 providers available... But the second i click History, it immediately pops back up and resumes.... Without me clicking anything

This bug has been fixed has now been fixed in the latest release. Thanks for reporting it! https://github.com/UnstoppableSwap/unstoppableswap-gui/releases/tag/v0.5.3

**Comment by Cokeman60** [comment]
Everything resolved? 
Do i dare ask what was wrong?

**Comment by binarybaron** [comment]
> Everything resolved?
> Do i dare ask what was wrong?

No, not everything resolved. Only the issue I quoted in my comment above.

**Comment by Cokeman60** [comment]
what else is wrong? Should i hold off on using tool longer? 
I only installed 0.5.3 i haven't tested yet.

**Comment by Cokeman60** [comment]
Tool still no work. I was hoping for Christmas miracle. I'm over it. Make a post when its officially live again. Thanks.
